src/components/TextInput/styles.tsx

```diff
@@ -29,6 +30,10 @@ export const InputText = styled(P)<{ $error: boolean }>`
   }
 `;
 
+export const InputTextLarge = styled(InputText)<{ $height?: number }>`
+  min-height: ${props => (props.$height ? props.$height : 0.875)}rem;
```

Comment:
should we use destructuring instead?

Reply:
actually we don't need this style anymore! but ty!